project/src/components/main-screen/main-screen.tsx

```diff
@@ -0,0 +1,138 @@
+            </div>
+
+            <div className="film-card__desc">
+              <h2 className="film-card__title">{title}</h2>
+              <p className="film-card__meta">
+                <span className="film-card__genre">{genre}</span>
+                <span className="film-card__year">{released}</span>
+              </p>
+
+              <div className="film-card__buttons">
+                <button className="btn btn--play film-card__button" type="button">
+                  <svg viewBox="0 0 19 19" width="19" height="19">
+                    <use xlinkHref="#play-s"></use>
+                  </svg>
+                  <span>Play</span>
+                </button>
+                <button className="btn btn--list film-card__button" type="button">
+                  <svg viewBox="0 0 19 20" width="19" height="20">
+                    <use xlinkHref="#add"></use>
+                  </svg>
+                  <span>My list</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <div className="page-content">
+        <section className="catalog">
+          <h2 className="catalog__title visually-hidden">Catalog</h2>
+
+          <ul className="catalog__genres-list">
+            <li className="catalog__genres-item catalog__genres-item--active">
+              <a href="#todo" className="catalog__genres-link">All genres</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Comedies</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Crime</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Documentary</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Dramas</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Horror</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Kids & Family</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Romance</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Sci-Fi</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#todo" className="catalog__genres-link">Thrillers</a>
+            </li>
+          </ul>
+
+          <div className="catalog__films-list">
+            {SMALL_CARD_FILMS_INFO.map((filmCard) => smallFilmCard(filmCard))}
```

Comment:
```suggestion
            {SMALL_CARD_FILMS_INFO.map(smallFilmCard)}
```

зачем еще одну переменную создавать, у тебя и так туда будут переданы нужные данные.